src/Oci8/Query/Grammars/OracleGrammar.php

```diff
@@ -203,7 +203,7 @@ protected function wrapValue($value)
             return $value;
         }
 
-        $value = $this->isReserved($value) ? Str::lower($value) : Str::upper($value);
+        $value = Str::upper($value);
```

Comment:
Anyways, I think the PR should be re-targeted to master and wrapValue should be force to upper case too?

```php
    protected function wrapValue($value)
    {
        if ($this->isReserved($value)) {
            return Str::upper(parent::wrapValue($value));
        }

        return $value !== '*' ? sprintf($this->wrapper, $value) : $value;
    }
```